### ISSUE TITLE
Friends List

### ISSUE THREAD

**Issue by tectal831** [EdenServer/community#465]
Friends List doesnt work

**Comment by Setz0r** [comment]
friends list is likely never going to work.

**Comment by godmode** [comment]
Let’s go ahead and leave it open as a reminder for awhile. We may get time to do it far in the future.

**Comment by Caelic** [comment]
Related topic: https://www.edenxi.com/forums/viewtopic.php?f=1&t=481&start=10

Basically, the FFXI client communicates with the PlayOnline servers to send and retrieve packets for the friend list, not the game servers it is currently playing on. The best bet is to have a plugin or the bootloader itself check all outgoing packets from the client so that it can redirect them to the current private server you're playing on instead of sending them to POL (to then be consequently tossed into the void by POL's servers). Incoming packets to the client may be harder, though? The packet catcher would need to take incoming friend list related packets from the game server and "spoof" them to appear to be from POL so that the client can read them properly.

There are a lot of extra details that need to be considered too because this would need to be done from the ground up. It's definitely possible. It's just such a monumental task.

**Comment by Setz0r** [comment]
yeah, we've already done all the research, it's just the level of effort of parsing and decrypting POL packets that's the problem.  I spent about a solid week on it a few weeks ago, only to come to the conclusion that it's too much work for something so stupidly pointless when things like discord are mainstream now.

**Comment by Caelic** [comment]
> yeah, we've already done all the research, it's just the level of effort of parsing and decrypting POL packets that's the problem. I spent about a solid week on it a few weeks ago, only to come to the conclusion that it's too much work for something so stupidly pointless when things like discord are mainstream now.

I figured. Thanks for the input and effort, Setz.
